### ISSUE TITLE
Rails 4

### ISSUE THREAD

**Issue by weblee** [leejarvis/hound#4]
Would be great if you can update this gem to be rails 4 compitable

**Comment by leejarvis** [comment]
I agree that's a great idea. I don't have a lot of time for this feature though, maybe you could take a shot at it?
